rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -29,219 +29,437 @@
 import rocal_pybind as b
 from amd.rocal.pipeline import Pipeline
 
-def blend(*inputs,ratio=None):
-    kwargs_pybind = {"input_image0":inputs[0], "input_image1":inputs[1], "is_output":False ,"ratio":ratio}
-    blend_image = b.Blend(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+
+def blend(*inputs, ratio=None, rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
+    """
```

Comment:
@SundarRajan28 we need to confirm with Kiriti how to pass API descriptions for Python in Doxygen so that it auto generates.

Reply:
@SundarRajan28 PLease follow this example and make changes accordingly: https://github.com/Feneric/doxypypy#example